cerebrium/endpoints/async.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "Asynchronous Execution (Preview)"
+description: "Execute calls to a Cerebrium app to be run asynchroously"
+---
+
+<Note>
+This is a new feature! As such, the API is still currently subject to changes.
+
+Most applications are expected to work with the current implementation.
+However, should you encounter an issue deploying a Custom Runtime please
+reach out to us on Discord!
+
+</Note>
+
+Many applications require asynchronous execution of functions in a "fire-and-forget" fashion
+in order to work optimally. In this scenario, you hand off execution of a function to be Cerebrium's
+responsibility, while you as the developer are responsible for ensuring that data is able leave the function.
+
+Through parameterizing your request to the Cerebrium API, you can enable your function to executed asynchronously
+by setting the `async` query parameter of your request to `true`. This would look something like this:
+
+```bash
+curl -X POST https://api.cortex.cerebrium.ai/v4/<YOUR-PROJECT-ID>/<YOUR-APP>/run?async=true'\
+       -H 'Content-Type: application/json'\
+       -H 'Authorization: Bearer <YOUR-JWT-TOKEN>\
+       --data '{"param": "hello world"}'
+```
+
+This would _immediately_ return a response akin to the following, with a body that only specifies a `run_id`:
+
+```bash
+HTTP/1.1 202 Accepted
+Connection: close
+Content-Length: 50
+Content-Type: application/json
+Date: Tue, 12 Nov 2024 03:13:25 GMT
+Server: envoy
+Vary: Origin
+X-Envoy-Upstream-Service-Time: 2
+X-Request-Id: 21eb3b98-4b10-9ad6-8681-a47172828024
+
+{"run_id":"21eb3b98-4b10-9ad6-8681-a47172828024"}
+```
+
+In the background, Cerebrium will now run this synchronous function for you. Note the term synchronous.
+Your function is still expected to behave **synchronously**. That is, it must open a connection, keep
+it open while it is doing work, then return a response once it is done. This effectively allows you
+to treat synchronous request-responses you write as asynchronous, and you can do anything within the scope
+of your call to the function. Functions executed asynchronously are also subject to a timeout of _12 hours_
+as opposed to the standard 15 minutes.
+
+In order to use async execution effectively, you must ensure that your function exports any relevant data you
+need, since you will no longer be able to receive a request. An easy way to achieve this would be to combine
+async execution with a specified `webhookEndpoint`, to have Cerebrium automatically forward the body of
+the function response once it has returned:
+
+```bash
+curl -X POST <https://api.cortex.cerebrium.ai/v4/><YOUR-PROJECT-ID>/<YOUR-APP>/run?async=true&webhookEndpoint=https%3A%2F%2Fwebhook.site%2F'\
+ -H 'Content-Type: application/json'\
+ -H 'Authorization: Bearer <YOUR-JWT-TOKEN>\
+ --data '{"param": "hello world"}'
```

Comment:
Maybe give a example of what the forwarded body would look like? So they can see it contains runtime, runId etc. Else might be better to refer to webhook endpoint doc